.gitignore

```diff
@@ -55,4 +55,5 @@ tilt-settings.json
 # e2e output
 test/e2e/junit.e2e_suite.*.xml
 test/e2e/logs/*
+test/e2eclusterctl/config/azure-dev-envsubst.yaml
```

Comment:
Are you keeping both `e2eclusterctl` and `e2e` side by side to be able to switch back? should we plan to replace e2e by e2eclusterctl at some point?

Reply:
I think we could directly switch to the clusterctl framework by removing `test/e2e` package and renaming `test/e2eclusterctl` to `test/e2e` since all the test cases in `test/e2e` have been converted.

Reply:
that sounds good to me, will be easier for maintenance and less confusing. If things go wrong we can always revert. Let's make it a separate commit to delete the old framework and rename the new one so it's easier to revert without deleting the new framework.